app/api-v2/routes/run-process.js

```diff
@@ -46,7 +46,7 @@ module.exports = function (apiService) {
           }
 
           if (`parent_index` in output) {
-            if (output.parent_index > request.inputs.length) {
+            if (output.parent_index < 0 || !(output.parent_index < request.inputs.length)) {
```

Comment:
This probably needs 2 new tests right?
1. `parent_index < 0`
2. `parent_index === inputs.length`